doc/configuration.md

```diff
@@ -1,56 +1,48 @@
 ## Configuration
 
-Amoc is configured through the OTP application environment variables that
-can be set in the `app.config` configuration file (see [rebar3 documentation](https://www.rebar3.org/docs/releases#section-overlays))
-or using the operating system environment variables (uppercase with prefix ``AMOC_``). Note that the operating system environment variables are evaluated as erlang code
+Amoc is configured through the environment variables (uppercase with prefix ``AMOC_``).
+Note that the environment variables are evaluated as erlang code
 
 Amoc supports the following generic configuration parameters:
 
 * ``nodes`` - required for the distributed scenario execution, a list of nodes that should be clustered together:
     * default value - empty list (`[]`)
-    * os env example: `AMOC_NODES="['amoc@amoc-1', 'amoc@amoc-2']"`
-    * app.config example:  `{nodes,  ['amoc@amoc-1', 'amoc@amoc-2]}`                                  
+    * example: `AMOC_NODES="['amoc@amoc-1', 'amoc@amoc-2']"`
 
 * ``api_port`` - a port for the amoc REST interfaces:
     * default value - 4000
-    * os env example: `AMOC_API_PORT="4000"`
-    * app.config example:  `{api_port, 4000}`
+    * example: `AMOC_API_PORT="4000"`
                                       
 * ``interarrival`` - a delay (in ms, for each node in the cluster independently) between creating the processes
   for two consecutive users:
     * default value - 50 ms.
-    * os env example: `AMOC_INTERARRIVAL="50"`
-    * app.config example:  `{interarrival, 50}`
+    * example: `AMOC_INTERARRIVAL="50"`
+    * this parameter can be updated runtime (in the same way as scenario configuration).
```

Comment:
at runtime

Reply:
i think "runtime" can be used as a flat adverb, but I don't mind changing it.

Reply:
I'm afraid runtime is a noun and it needs the particle